src/Fields/Select/index.js

```diff
@@ -86,6 +86,7 @@ const Select = ({
       onChange={onChange}
       as={
         <ReactSelect
+          data-testid='question-select'
```

Comment:
It is supposed for the perfect practise trying to avoid the data-testid, It is supposed that we should be able to test everything without making modifications. But for me it is fine and necessary sometimes and Isma use it too so it is fine. It is just a curious data